### DISCUSSION TITLE
Add chapter location navigation bar on the page

### DISCUSSION THREAD

**Discussion by Sunny19191** [web-infra-dev/rspress#1218] [Ideas]
像Docusaurus一样，可以展示章节所在位置
![导航](https://github.com/web-infra-dev/rspress/assets/171238222/a976240c-0291-4ffa-b3cf-1b17c3dd7781)

**Comment by Timeless0911** [comment]
That's a great idea. The breadcrumb component can be made optional via a plugin, like what [@rspress/plugin-last-updated](https://rspress.dev/plugin/official-plugins/last-updated#rspressplugin-last-updated) do. We are welcome for contributors to add such a feature.
